### ISSUE TITLE
Provide a end to end fine tune notebook.

### ISSUE THREAD

**Issue by Imran-tech-lab** [facebookresearch/nougat#56]
Hi, I want to experiment, and fine tune nougat on other language. Please provide a notebook for fine tuning.

**Comment by marwinsteiner** [comment]
Check the readme, there are some instructions on how to fine-tune. You can just make the Jupyter notebook yourself...?

**Comment by marwinsteiner** [comment]
No, you have to make it yourself according to the instructions in the readme.

**Comment by marwinsteiner** [comment]
I am working on a little dataset generator for my own Nougat fine-tuning project. You have to create your own dataset for fine-tuning. I don't think they plan on generating a dataset for you... would be nice though ;). 
I would actually appreciate some more detail in the dataset generation section of the readme. I want to know why each step is being done and what is happening, not just step-by-step, and I guess @imrankh46, you would appreciate this as well. 

Eventually Lukas or one of the other collaborators might give an update concerning fine-tuning so that might be worth waiting for, especially regarding  https://github.com/facebookresearch/nougat/issues/52#issuecomment-1713945716. Then we'll know more regarding feasibility and concrete how-to steps I would hope.

It also always helps to read the paper/have a copy on hand. PDF: https://arxiv.org/abs/2308.13418

**Comment by nullhook** [comment]
> I don't think they plan on generating a dataset for you... would be nice though

I'm confused, aren't there helpers inside `/nougat/dataset/` that does this for you? This [section](https://github.com/facebookresearch/nougat#dataset) also says how

**Comment by NielsRogge** [comment]
Hi,

See also #128 with some tutorial notebooks.
